templates/bamboo/0/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+bamboo-data:
+    image: busybox
```

Comment:
Hmm, I didn't have such a problem when testing it on my rancher config. I'll check it and let you know.